plugins/quota/validate.go

```diff
@@ -3,38 +3,65 @@ package quota
 import (
 	"fmt"
 
+	daemon "github.com/gluster/glusterd2/glusterd2/daemon"
 	"github.com/gluster/glusterd2/glusterd2/volume"
 	"github.com/gluster/glusterd2/glusterd2/xlator"
+	"github.com/gluster/glusterd2/pkg/errors"
 )
 
 const (
-	quotaEnabledKey = "features.quota"
+	quotaEnabledKey = "quota.enable"
 )
 
 func validateOptions(v *volume.Volinfo, key string, value string) error {
```

Comment:
suggestion:
same here
we can change 
```
key string, value string
```
to 
```
key,value string
```

Reply:
done